src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+import org.kuali.kfs.gl.batch.CollectorBatch;
+import org.kuali.kfs.sys.service.UniversityDateService;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractFile;
+import edu.cornell.kfs.concur.batch.service.ConcurStandardAccountingExtractCreateCollectorFileService;
+import edu.cornell.kfs.concur.batch.service.ConcurStandardAccountingExtractValidationService;
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
+
+public class ConcurStandardAccountingExtractCreateCollectorFileServiceImpl
+        implements ConcurStandardAccountingExtractCreateCollectorFileService {
+
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(
+            ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.class);
+
+    protected ConcurStandardAccountingExtractValidationService concurSAEValidationService;
+    protected UniversityDateService universityDateService;
+    protected DateTimeService dateTimeService;
+    protected ParameterService parameterService;
+
+    @Override
+    public boolean buildCollectorFile(ConcurStandardAccountingExtractFile saeFileContents) {
+        ConcurStandardAccountingExtractCollectorBatchBuilder builder = new ConcurStandardAccountingExtractCollectorBatchBuilder(
+                universityDateService::getFiscalYear, dateTimeService::toString,

```

Comment:
would it be better to define these lamba functions as a variables to help with understanding what is going on?